svthalia/concrexit
# Refactor the information check emails to require only a single translation

Closes #1068

Refactor the information check emails to require only a single translation

### How to test
Steps to test the changes you made:
1. Run `./manage.py sendinformationcheck`
2. Check the output

Here is a sample: 
[email_output.txt](https://github.com/svthalia/concrexit/files/4606474/email_output.txt)
